### PR TITLE
Closes #6 - Create download and install commands

### LINKED ISSUE #6: Create download and install commands
It would be nice if fluo-dev could automate the download and install of dependencies.  The download command could also download hashes and signature for users to verify releases.

### PR DESCRIPTION
**Pull request by mikewalch**
Pull request for #6

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,22 +15,18 @@ First, clone the fluo-dev repo on a local disk with enough space to run Hadoop,
...
+Create `env.sh` from the example.  This file is used to configure fluo-dev for your enviornment.
 ```
 cd conf/
 cp env.sh.example env.sh
 vim env.sh
 ```
 
+If you have not already, clone the Fluo repo.  Set FLUO_REPO in your env.sh to this location.
+```
+git clone git@github.com:fluo-io/fluo.git
```
Comment (keith-turner) [inline README.md]: Not positive, but I think cloning that way uses ssh and requires a GH account.  Could use https for cloning which would not require a GH account.

File: README.md [hunk 2]
```diff
@@ -44,10 +40,25 @@ every time, add the following alias to your `~/.bashrc`:
 alias fluo='fluo-dev fluo'
 ```
 
+Next, you need to download the binary tarballs of Fluo dependencies (i.e Accumulo, Hadoop, and
```
Comment (keith-turner) [inline README.md]: I feel like there should be a new heading in the document here, maybe a Running or Using section.  Currently this falls under the Installation heading.

File: README.md [hunk 3]
```diff
@@ -44,10 +40,25 @@ every time, add the following alias to your `~/.bashrc`:
...
+Zookeeper).  This can be done by running the command below which downloads all tarballs and their
+corresponding hashes and signatures to the directory specified by DOWNLOADS in your env.sh. It 
+will use Apache download mirror specified by APACHE_MIRROR in env.sh.  Other mirrors can
```
Comment (keith-turner) [inline README.md]: Did you think about making these vars from env fixed width like `APACHE_MIRROR`?

File: README.md (hunk 1 above)
Comment (mikewalch) [inline README.md]: fixed in bb5865548e0

File: README.md (hunk 2 above)
Comment (mikewalch) [inline README.md]: fixed in bb5865548e0

File: README.md (hunk 3 above)
Comment (mikewalch) [inline README.md]: fixed in bb5865548e0

File: README.md [hunk 4]
```diff
@@ -44,16 +40,35 @@ every time, add the following alias to your `~/.bashrc`:
...
+Running Fluo using fluo-dev
+---------------------------
 
+With fluo-dev set up, you can now use it to download, configure, and run Fluo and its dependencies.
+
+First, run the command below to download the binary tarballs of Fluo dependencies (i.e Accumulo, Hadoop, 
+and Zookeeper). The command below downloads all tarballs and their corresponding file hashes and 
+signatures to the directory specified by `DOWNLOADS` in your env.sh. It will use the Apache download 
+mirror specified by `APACHE_MIRROR` in env.sh.  Other mirrors can be chosen from [this website][1].
+This command will also help you verify checksums and signatures (if you have `gpg` installed).
```
Comment (keith-turner) [inline README.md]: Could point out that its important to inspect output related to hashes before proceeding.

Comment (mikewalch) [inline README.md]: improved in 98baeff1eb

**Pull request merged**